bokeh/server/tests/test_server.py

```diff
@@ -167,7 +167,7 @@ def test__lifecycle_hooks():
     application = Application()
     handler = HookTestHandler()
     application.add(handler)
-    with ManagedServerLoop(application, check_unused_sessions_milliseconds=20) as server:
+    with ManagedServerLoop(application, check_unused_sessions_milliseconds=30) as server:
```

Comment:
it doesn't look like this timeout was "the point" of the test so it should be something high enough to work reliably. I guess the tradeoff is as you make it higher the test takes longer to run. But 10ms is not going to matter there.